core/src/main/java/nl/nn/adapterframework/scheduler/job/SendMessageJob.java

```diff
@@ -61,7 +61,7 @@ public void execute() throws JobExecutionException, TimeoutException {
 			//sendMessage message cannot be NULL
 			Message message = new Message((getMessage()==null) ? "" : getMessage());
 			PipeLineSession session = new PipeLineSession();
-			session.put(PipeLineSession.messageIdKey, Misc.createSimpleUUID()); //Create a dummy messageId so the localSender uses it as correlationId for the calling adapter.
+			session.put(PipeLineSession.correlationIdKey, Misc.createSimpleUUID()); //Create a dummy messageId so the localSender uses it as correlationId for the calling adapter.
```

Comment:
Dit bestaat alleen om er voor te zorgen dat in de ladybug niet 2 rapporten komen. De aanroepende adapter moet in/genest zitten in de job (het buitenste of eerste rapport).
Misschien handig dit als commentaartje er bij te zetten (en ook even te testen!).

Reply:
getest, en het werkt nog steeds